.github/workflows/cleanup.yml

```diff
@@ -65,4 +65,32 @@ jobs:
               --name "$account" \
               --resource-group "$RESOURCE_GROUP" \
               --yes
+          done
+
+  cleanup-akv:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Log into Azure
+        uses: azure/login@v2
+        with:
+          client-id: ${{ secrets.MANAGED_ID_CLIENT_ID }}
+          tenant-id: ${{ secrets.MANAGED_ID_TENANT_ID }}
+          subscription-id: 7ca35580-fc67-469c-91a7-68b38569ca6e
```

Comment:
we already have a var for subscription.

Reply:
It has a different value since we need to temporarily use `Parakeet-Services-NonProd` which allows us to deploy storage accounts with shared key access until our exception is granted